packages/wonder-blocks-modal/docs.md

```diff
@@ -1,10 +1,29 @@
 Looking for docs for StandardModal, OneColumnModal, or TwoColumnModal click [here](https://deploy-preview-389--wonder-blocks.netlify.com/#modal)
 
-## Wrapping Modals
+## Accessibility
 
-Often you'll want to define a new modal component by wrapping an existing
-modal component.  These wrapped components will also work with ModalLauncher
-in the same way the default ones do.
+It should follow guidelines from [W3C](https://www.w3.org/TR/wai-aria-practices/#dialog_modal).
+
+### Keyboard Interaction
+When a dialog opens, focus moves to an element inside the dialog. See notes below regarding initial focus placement.
+
+- Tab:
+    - Moves focus to the next tabbable element inside the dialog.
+    - If focus is on the last tabbable element inside the dialog, moves focus to the first tabbable element inside the dialog.
+- Shift + Tab:
+    - Moves focus to the previous tabbable element inside the dialog.
+    - If focus is on the first tabbable element inside the dialog, moves focus to the last tabbable element inside the dialog.
+- Escape: Closes the dialog.
+
+### WAI-ARIA Roles
+- The element that serves as the **dialog container** has `aria-role` defined as `dialog`.
+- The dialog has a value set for the `aria-labelledby` property that refers to a **visible dialog title**.
+
+## Wrapping Dialogs
```

Comment:
I got some feedback that this example is too complex to start with and should be moved down.  Unfortunately, styleguidist makes a lot of decisions about where things live.  I would like to do something about it in future but don't worry about it right now.